server/src/main/java/org/elasticsearch/action/synonyms/TransportGetSynonymsAction.java

```diff
@@ -30,7 +30,7 @@ public TransportGetSynonymsAction(TransportService transportService, ActionFilte
 
     @Override
     protected void doExecute(Task task, GetSynonymsAction.Request request, ActionListener<GetSynonymsAction.Response> listener) {
-        synonymsManagementAPIService.getSynonymRules(
+        synonymsManagementAPIService.getSynonymSetRules(
```

Comment:
I figured changing this name made the intention clearer, as we now have synonym set rules and single synonym rule